src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Tests/Utils/SqidEncoderTests.cs

```diff
@@ -0,0 +1,130 @@
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Utils;
+using Sqids;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Tests.Utils;
```

Comment:
Would put this in the `Public.Data.Utils.Tests` project

Reply:
I forgot that existed! Haha